modules/nodalsAiRtdProvider.js

```diff
@@ -246,11 +246,11 @@ class NodalsAiRtdProvider {
    */
 
   #hasRequiredUserConsent(userConsent) {
-    if (userConsent?.gdpr?.gdprApplies !== true) {
+    if (userConsent?.gdpr?.gdprApplies === false) {
```

Comment:
what happens when the tcf module doesnt exist?

Reply:
If the TCF module is responsible for creating the gdpr object (or the properties within), then this check will fail so we will move onto the next set of checks.

This check is in place to quickly return true if the user or property is not within gdpr jurisdiction.